azure-pipelines.yml

```diff
@@ -103,10 +132,7 @@ strategy:
     #   image: "vs2017-win2016"
     #   ghc-flavor: "da-ghc-8.8.1"
     #   resolver: "nightly-2019-09-26"
-    mac-da-ghc-881-881:
-      image: "macOS-10.13"
-      ghc-flavor: "da-ghc-8.8.1"
-      resolver: "nightly-2019-09-26"
+    # ---- ghc-flavor : da-ghc-8.10.1 (doesn't exist)
 
```

Comment:
Why is this one removed?

Reply:
Argh. Just a mistake. Nice catch and thank-you. I don't know why but I struggle with that YAML format and keeping this matrix straight!